openlibrary/templates/type/author/edit.html

```diff
@@ -132,7 +135,7 @@ <h1>$_("Edit Author")</h1>
             </div>
 
         $if ctx.user and ctx.user.is_admin():
-            <div class="adminOnly" style="position:absolute;top:50px;right:20px;"><button type="submit" value="$_('Delete Record')" name="_delete" title="$_('Delete this macro?')" id="deleteTop">$_("Delete Record")</button></div>
+            <div class="adminOnly" style="position:absolute;top:50px;right:20px;"><button type="submit" value="$_('Delete Record')" name="_delete" title="$_('Delete Record')" id="deleteTop">$_("Delete Record")</button></div>
```

Comment:
note to self: We should grep for inline styles in/as a different issue.

Reply:
I noticed a few while doing this, and noticed at least one you squished when I rebased. Agree that it would be a great separate task to tidy up UI and CSS!